Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceEnrollmentStatusPageWindows10/MSFT_IntuneDeviceEnrollmentStatusPageWindows10.psm1

```diff
@@ -9,7 +9,7 @@ function Get-TargetResource
         [System.String]
         $DisplayName,
 
-        [Parameter()]
+        [Parameter(Mandatory = $true)]
```

Comment:
Please keep the $Id as non-mandatory, Nik changed this for several resources on #3681 to ease the apply of policies on different tenants. Additionally the property should also change from Required to Write in its schema.

Reply:
I was getting a validation error from the DSC engine when the key was not marked as mandatory 
Id is marked as Key is the schema

Reply:
![image](https://github.com/microsoft/Microsoft365DSC/assets/79221284/e76d4335-cf13-4e3a-a3b2-03580ce9587a)

Reply:
Yep, as I mention above you need to change the schema to have *Id* as Write instead of Required otherwise you'll get that error message.

Reply:
Fair enough just caught that - pushing that change now